tests/python-tests/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+# Usage: run_tests.sh <maat_python_module.so> <sleigh_spec_dir>
+# Copy maat python module locally
+cp $1 .
+export MAAT_SLEIGH_DIR=$2
+# Run python tests
+python3 -m pytest .
+status=$?
+# Remove local copy of python module
+rm ./$(basename $1)
+exit $status
```

Comment:
Nice, that'd be way more elegant than the shell script trick!

Reply:
The environment needs to be a list: 
`ENVIRONMENT "PYTHONPATH=$<TARGET_FILE_DIR:maat_python>; MAAT_SLEIGH_DIR=${spec_out_dir}"`
Otherwise it seems to work like a charm!